stellarphot/settings/models.py

```diff
@@ -68,6 +72,9 @@ class BaseModelWithTableRep(BaseModel):
     Class to add to a pydantic model YAML serialization to an Astropy table.
     """
 
+    # NOTE WELL that this will set the configuration for all subclasses of this
```

Comment:
OK, Not sure what the point of "NOTE WELL" is here.  Do you really want me to pay attention that this sets the configuration of all subclasses of `BaseModelWithTableRep`?

Reply:
Ah